### PR TITLE
IT-870 - setup JC IDP 

### PR DESCRIPTION
**Pull request by xschildw**
[IT-870], sorry about single PR...
Ignore the CMKTemplate.json, it's not automatically run.

[IT-870]: https://sagebionetworks.jira.com/browse/IT-870

### CODE REVIEW

File: templates/jumpcloud-idp.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      Name: "synapseprod-developer"
+      Metadata: !Ref SynapseProdDeveloperMetadata
+      URL: ""
+  SynapseProdDeveloperSamlProviderRole:
+    Type: AWS::IAM::Role
+    Properties:
+      MaxSessionDuration: 28800
+      RoleName: !GetAtt SynapseProdDeveloperSamlProvider.Name
+      ManagedPolicyArns:
+        - arn:aws:iam::aws:policy/PowerUserAccess
```
Comment (zaro0508) [inline templates/jumpcloud-idp.yaml:50]: are you sure you want to give PowerUserAccess to developer role in your prod account?

Comment (xschildw) [reply templates/jumpcloud-idp.yaml:50]: Correct, that's too much. Now they have ReadOnly++... What did you do in scicomp/bridge? RO and then move users to admin if/when they need more perms?

Comment (zaro0508) [reply templates/jumpcloud-idp.yaml:50]: For Bridge I only setup a `bridge-admin` role.  I will add other roles as needed.  
https://github.com/Sage-Bionetworks/Bridge-infra/blob/develop/templates/prod/jumpcloud-idp.yaml#L22 

Adding  RO, DEV and ADMIN roles then moving people between roles seems like a sound strategy to me.

Comment (zaro0508) [reply templates/jumpcloud-idp.yaml:50]: Check my PRs for Bridge and Admincental to see how to setup shared policies for roles and service accounts.

Comment (xschildw) [reply templates/jumpcloud-idp.yaml:53]: Same principle as yours but RO, DEV (poweruser), ADM. Nobody should be in DEV unless needed for particular task.

**Review by zaro0508 — APPROVED** [review]
looks good.

**Pull request merged**